starter-code/scripts/article.js

```diff
@@ -13,6 +13,9 @@ Article.prototype.toHtml = function() {
   // TODO: Use handlebars to render your articles!
   //       - Select your template from the DOM.
   //       - Now "compile" your template with Handlebars.
+  var source = $('#template').html();
+  var template = Handlebars.compile(source);
+  var html = template(this);
```

Comment:
template(this) is called too soon. Your handlebars template expects a value for "publishStatus" but that property gets assigned below this line, so it never gets rendered to the template.